invenio_app_ils/ui/src/invenio_app_ils/common/api/urls.js

```diff
@@ -0,0 +1,22 @@
+import { generatePath } from 'react-router-dom';
+
+const ApiBase = '/api';
```

Comment:
that's cool idea, maybe we make it for all the API urls ?

Reply:
We could :)